cabal.project

```diff
@@ -2,11 +2,11 @@ packages: pandoc.cabal
 
 package pandoc
   flags: +embed_data_files -trypandoc
-  ghc-options: -fhide-source-paths -j +RTS -A32M -RTS
+  ghc-options: -j +RTS -A32M -RTS
 
 package pandoc-citeproc
   flags: +embed_data_files +bibutils -unicode_collation -test_citeproc -debug
-  ghc-options: -fhide-source-paths -j +RTS -A32M -RTS
+  ghc-options: -j +RTS -A32M -RTS
 
```

Comment:
Why not just add `-fhide-source-paths` in pandoc.cabal, conditionally on ghc version?
That would eliminate the need for an extra makefile target and a cabal.project.local.
And (I think) people could still override this default by adding `-fno-hide-source-paths` in their cabal.project.local (though I haven't tested this).

Reply:
I was afraid that this would be too opinionated and wouldn't quite belong in a cabal file. I'll happily change it.

I realized only yesterday that `cabal new-configure` overwrites cabal.project.local, which makes my approach even worse.

Reply:
I understand your thinking here -- but I think being a bit opinionated is okay if it lets us avoid some complexity.